crates/sui-replay/src/lib.rs

```diff
@@ -633,3 +643,18 @@ pub(crate) fn chain_from_chain_id(chain: &str) -> Chain {
         Chain::Unknown
     }
 }
+
+fn network_from_url(url: Option<String>) -> String {
```

Comment:
@laura-makdah maybe we could return the URL as is? the `ExplorerLink` will work with urls too. Or do we have a "standard" where all RPC urls contain the network `keyword`?

Reply:
Passing the url straight to the ExplorerLink worked!